src/components/sequencing/Sequences.svelte

```diff
@@ -59,6 +63,41 @@
     const workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID);
     goto(`${base}/sequencing/new${workspaceId ? `?${SearchParameters.WORKSPACE_ID}=${workspaceId}` : ''}`);
   }
+
+  async function importLibarySequences(): Promise<void> {
+    const workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID);
+    if (workspaceId === null) {
+      console.log("Workspace doesn't exist");
```

Comment:
Our current convention for most complicated modals is to call them from an effects function as well as showing toasts. Would you mind moving this into one?

Reply:
Ahhh, got it. I moved part of this function to the `effects.ts`